e2e-tests/playwright/lib/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@mattermost/playwright-lib",
+  "version": "10.6.0",
+  "type": "commonjs",
+  "files": [
+    "dist"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "require": "./dist/index.js"
+    }
+  },
+  "keywords": [
+    "mattermost",
+    "e2e",
+    "playwright",
+    "test-automation"
+  ],
+  "scripts": {
+    "build": "rollup -c --bundleConfigAsCjs",
+    "build:watch": "npm run build -- --watch",
+    "build-tsc": "tsc --build --verbose",
+    "build-tsc:watch": "tsc --watch --preserveWatchOutput",
+    "clean": "rm -rf dist node_modules *.tsbuildinfo",
+    "tsc": "tsc -b"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@axe-core/playwright": "4.10.1",
+    "@percy/cli": "1.30.7",
+    "@percy/playwright": "1.0.7",
+    "async-wait-until": "2.0.23",
+    "axe-core": "4.10.2",
+    "deepmerge": "4.3.1",
+    "dotenv": "16.4.7",
+    "mime-types": "2.1.35",
+    "@mattermost/client": "10.6.0-0",
+    "@mattermost/types": "10.6.0-0",
+    "uuid": "11.0.5"
+  },
+  "devDependencies": {
+    "@rollup/plugin-typescript": "12.1.2",
+    "@types/mime-types": "2.1.4",
+    "@types/node": "22.13.1",
+    "@types/react": "19.0.8",
+    "rollup": "4.34.6",
+    "rollup-plugin-copy": "3.5.0"
+  },
+  "peerDependencies": {
+    "@playwright/test": "^1.50.1",
+    "@mattermost/client": "10.6.x",
+    "@mattermost/types": "10.6.x"
+  },
+  "peerDependenciesMeta": {
+    "@playwright/test": {
+      "optional": false
```

Comment:
Minor nit, setting optional to false is optional 😛

Reply:
Done, removed :)